### ISSUE TITLE
Nothing shows when "VillagerHelper Enabled"

### ISSUE THREAD

**Issue by shouxd** [Ivan-1F/VillagerHelper#1]

**Comment by Ivan-1F** [comment]
Are you playing on a server? This mod may not work in servers. Client mods can't get entity data from the server.
But I'm trying to develop a server side mod to send data to clients.

**Comment by Ivan-1F** [comment]
I have just released v1.2.0.
Install this mod on **both** client and server side and everything should work.

**Comment by shouxd** [comment]
Hi! I used version 1.2.0 on both server and client but other than it still not working, I got an unbearable fps loss and allocated memory went up to 100% (all of this on client). Server ran as always. Could this be a possible memory leak?

PD: I joined the server in a clear area where there were no villagers around

**Comment by Ivan-1F** [comment]
Can you upload your Minecraft version, log file and your mod list pls? It works will for me on both 1.15.2 and 1.16.4.

**Comment by shouxd** [comment]
Sure. Here they are:
[latestlog+mods.zip](https://github.com/Ivan-YFw/VillagerHelper/files/5859649/latestlog%2Bmods.zip)

**Comment by Ivan-1F** [comment]
Try 1.3.0. It should work.

**Comment by shouxd** [comment]
Hi again! I just tried new version and I couldnt even turn it on in VillagerHelpers config because my ram instantly went up in the sky when I joined the server which caused minecraft to cease responding and I had to kill the process from the task manager.

I tried again but this time enabling it from the initial game menu and realized that the "Done" button in VillagerHelper config" doesnt work. Also, when I lowered the render distance in "VillagerHelpers config" these two crashes happened, making it impossible to join the server and test if mod works:
[crash-2021-01-23_03.10.01-client.zip](https://github.com/Ivan-YFw/VillagerHelper/files/5859907/crash-2021-01-23_03.10.01-client.zip)

PD: Server keeps working as usual. I've not seen abnormal ram usage there atm
